package.json

```diff
@@ -3,62 +3,62 @@
   "scripts": {
     "dev": "yarn run development",
     "development": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --hide-modules --config=webpack.config.js",
-    "watch": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --watch --progress --hide-modules --config=webpack.config.js",
-    "watch-poll": "yarn run watch -- --watch-poll",
+    "watch": "yarn run development --watch",
+    "watch-poll": "yarn run watch --watch-poll",
     "hot": "cross-env NODE_ENV=development node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline --hot --config=webpack.config.js",
     "prod": "yarn run production",
     "production": "cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --progress --hide-modules --config=webpack.config.js"
   },
   "dependencies": {
-    "@fortawesome/fontawesome-free-webfonts": "^1.0.6",
+    "@fortawesome/fontawesome-free": "^5.6.1",
     "@types/es6-promise": "^3.3.0",
-    "@types/react": "16.4.1",
-    "babel-preset-es2015": "^6.3.13",
+    "@types/jquery": "^3.3.28",
+    "@types/react": "16.7.17",
```

Comment:
the version of these type definitions should probably match the libraries they're for